src/angular-app/languageforge/lexicon/shared/fit-text.directive.ts

```diff
@@ -0,0 +1,33 @@
+import * as angular from 'angular';
+
+export class FitTextDirective implements angular.IDirective {
```

Comment:
You wouldn't have known this, but a while back we converted all of our directives to components in hopes that the code would be more compatible with a future upgrade to a new version of Angular.  I don't necessarily think that's a project goal anymore (upgrade to Angular) but I would prefer that we only write Components going forward instead of Directives.  Not a big deal, and since this code works - I'm fine with it as is.

Reply:
Attribute directives are not just an Angularjs concept and are still appropriate in [Angular ](https://angular.io/guide/attribute-directives) for manipulating dom elements. In this case changing it to a component would now also require changes to e2e. But It can be changed if you require.